client/packages/system/src/Location/ListView/Toolbar.tsx

```diff
@@ -48,16 +48,22 @@ export const Toolbar: FC<{
             }
           });
         })
-      ).then(() => {
-        setDeleteErrors(errors);
-        if (errors.length === 0) {
-          const deletedMessage = t('messages.deleted-locations', {
-            count: numberSelected,
-          });
-          const successSnack = success(deletedMessage);
-          successSnack();
-        }
-      });
+      )
+        .then(() => {
+          setDeleteErrors(errors);
+          if (errors.length === 0) {
+            const deletedMessage = t('messages.deleted-locations', {
+              count: numberSelected,
+            });
+            const successSnack = success(deletedMessage);
+            successSnack();
+          }
+        })
+        .catch(_ =>
```

Comment:
if all of the delete promises fail, then an error is thrown - and this was not being caught